### ISSUE TITLE
Super + number is not working

### ISSUE THREAD

**Issue by adeleglise** [instantOS/instantOS#20]
I was acustomed to use Super+[1-9] to change tag, however it's not working right now in instantWM. NOr Super+s to show the icons instead of tags numbers.

Am I missing something ? I've read https://instantos.github.io/instantos.github.io/youtube/hotkeys

**Comment by paperbenni** [comment]
The super key is the one with the window icon on it. It is the CMD key on a mac keyboard.

**Comment by adeleglise** [comment]
Yup I've always used the windows key as super (on I3 or AwesomeWM). 
But here, Super+number is not working. Nor Super+s.

**Comment by paperbenni** [comment]
Do other super + key combinations work?

**Comment by adeleglise** [comment]
Yes, all the other combinations works. I have to add that I'm using an Azerty keyboard.

**Comment by paperbenni** [comment]
I'm also using one. Do you have anything like simple x hotkey daemon running? What happens if you change keyboard layout? (You can do that in the system settings)

**Comment by adeleglise** [comment]
No special hotkey daemon running. I will try to change keyboard layout.

**Comment by Albynton** [comment]
I've got the same issue as adeleglise. It is related to the French keyboard layout: changing the layout to another (like US) makes the `Super` + Number work as it should.
It may be caused by the fact that AZERTY keyboards access the numbers with `Shift` (e.g. to type `3` without using the numpad, I do `Shift`+`&`).
